includes/modules/shipping_estimator.php

```diff
@@ -150,10 +151,10 @@ function shipincart_submit(){
   }
   // begin shipping cost
   if(!$free_shipping && $_SESSION['cart']->get_content_type() !== 'virtual'){
-    if (zen_not_null($_POST['scid'])){
+    if (isset($_POST['scid']) && zen_not_null($_POST['scid'])){
```

Comment:
FYI - I've also been converting "isset && zen_not_null" to `empty`, unless I'm specifically expecting a `falsy` value, which is rare.